assets/scss/modules/_forms.scss

```diff
@@ -667,4 +682,82 @@
       }
     }
   }
+
+  /* input radio */
+
+  .dp-input--radio {
+    margin: $dp-spaces--lv1;
+    display: inline-block;
+    max-width: 170px;
+    height: 30px;
+    position: relative;
+
+    label {
+      display: flex;
+      cursor: pointer;
+      font-weight: 500;
+      position: relative;
+      overflow: hidden;
+      outline: 0;
+      border: 1px solid transparent;
+      box-shadow: inset 0 0 0 2px transparent;
+
+      &:focus-within {
+        outline: 0;
+        border: 1px solid $dp-color-orange-focus;
+        box-shadow: inset 0 0 0 2px $dp-color-orange-focus;
+      }
+
+      input {
+        position: absolute;
+        left: -9999px;
+
+        &:checked + span {
+          &:before {
+            box-shadow: inset 0 0 0 7px $dp-color-green;
+          }
+        }
+
+        &:disabled + span {
+          &:before {
+            outline: 0;
+            border: 1px solid $dp-color-snow;
+            box-shadow: inset 0 0 0 4px $dp-color-snow;
+          }
+        }
+      }
+
+      .dp-disabled-checked {
```

Comment:
No convendría aprovechar la property `checked` del input del radio button? Entonces al definir la regla podriamos tener algo asi: `input:checked:disabled`?
Y luego, para el que tiene que usarlo, no tendria que considerar ponerle la clase `dp-disabled-checked` al input que esta checkeado.
Que te parece?

Reply:
❤️ Excellent comment !! thanks for the suggestion